lib/sync/index.js

```diff
@@ -121,24 +123,52 @@ function syncRecords(dataset_id, params, cb) {
   return cb(null, {});
 }
 
+
+/**
+ * Connect to mongodb & redis with the given connection urls.
+ * This should be called before `start()` is called
+ * 
+ * @param {string} mongoDBConnectionUrl A MongoDB connection uril in a format supported by the `mongodb` module
+ * @param {string} redisUrl A redis connection url in a format supported by the `redis` module
+ * @param {function} cb 
+ */
 function connect(mongoDBConnectionUrl, redisUrl, cb) {
-  async.parallel([
+  if (arguments.length < 3) throw new Error('connect requires 3 arguments');
+  
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)